examples/resolvers-planetscale/grafbase/resolvers/products/create.ts

```diff
@@ -0,0 +1,48 @@
+import { connect, cast } from "@planetscale/database";
+
+const config = {
+  host: process.env.DATABASE_HOST,
+  username: process.env.DATABASE_USERNAME,
+  password: process.env.DATABASE_PASSWORD,
+};
+
+const conn = connect(config);
+
+export const options = {
+  cast(field, value) {
+    switch (field.name) {
+      case "onSale": {
+        return Boolean(value);
+      }
+      default: {
+        return cast(field, value);
+      }
+    }
+  },
+};
+
+export default async function ProductsCreate(_, { input }) {
+  const { name, slug, price, onSale } = input;
+
+  // Check for unique constraint and error when supported
+
+  try {
+    const { insertId } = await conn.execute(
```

Comment:
thoughts on using Kelsey or Drizzle since it's very popular these days? That way we SEO rank for that too.

Reply:
I've got a mention of this in the guide and will create a separate guide for doing so, that way we have more content to produce :)